src/Polysemy/Final/More.hs

```diff
@@ -0,0 +1,32 @@
+module Polysemy.Final.More where
```

Comment:
Not fond of this module name. Suggestions for better one?

Reply:
Do we need these?

Reply:
Strictly, no, but `runFinalSem`, at the very least, does express some power which can't be achieved using the existing interpreters of `Final`. I'd like to keep that one, at least, because it's plausible that it's usable.

Besides, this _is_ `polysemy-zoo`.